site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
-- For an annotation task (independent images).
-  If an object exists on overlapped segments, the overlap is greater than zero
-  and the annotation is good enough on adjacent segments, it will be automatically merged into one object.
-  If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
-  you will have several bounding boxes for the same object.
-  Thus, you annotate an object on the first segment.
-  You annotate the same object on second segment, and if you do it right, you
-  will have one track inside the annotations.
-  If annotations on different segments (on overlapped frames)
-  are very different, you will have two shapes for the same object.
-  This functionality works only for bounding boxes.
-  Polygons, polylines, points don't support automatic merge on overlapped segments
-  even the overlap parameter isn't zero and match between corresponding shapes on adjacent segments is perfect.
+For example, you have a label for `face` and want to
+specify the type of face. Instead of creating additional
+labels for `male` and `female`, you can use attributes
+to add this information.
 
-### Segment size
+There are two types of attributes:
 
-Use this option to divide a huge dataset into a few smaller segments.
-For example, one job cannot be annotated by several labelers (it isn't supported).
-Thus using "segment size" you can create several jobs for the same annotation task.
-It will help you to parallel data annotation process.
+- **Immutable** attributes are unique and do not change from frame to frame.
+  For example, `age`, `gender`, and `color`.
+- **Mutable** attributes are temporary and can change from frame to frame.
+  For example, `pose`, `quality`, and `truncated`.
 
-### Start frame
+Added attributes will be available from the **Objects menu**:
 
-Frame from which video in task begins.
+![Attributes](/images/attributes.jpg)
 
-### Stop frame
+To add an attribute, do the following:
 
-Frame on which video in task ends.
+1. Go to the **Constructor** tab and click **Add attribute**.
 
-### Frame Step
+   ![Attributes](/images/attributes_01.png)
 
-Use this option to filter video frames.
-For example, enter `25` to leave every twenty fifth frame in the video or every twenty fifth image.
+2. In the **Name** field enter the name of the attribute.
+3. From the drop-down, select way to display the attribute in the **Objects menu**:
+   - `Radio` enables the selection of one option from several options.
+   - `Checkbox` enables the selection of multiple options.
+   - `Text` sets the attribute to a text field.
+   - `Number` sets the attribute to numerical field in the following format: `min;max;step`.
+4. In the **Attribute values** field, add attribute values. <br>To separate values use **Enter**.
+   <br>To delete value, use **Backspace** or click **x** next to the value name.
+5. (Optional) For mutable attributes, select **Mutable**.
```

Comment:
We have a special value for an attribute `__undefined__`. Could you please document it as well?